welcome.yaml

```diff
@@ -8,10 +8,12 @@
 
       description: >
         This is the official Discord server for the HelixLauncher project.
+
+        This is not a dead project. Most of the devs are currently busy with other projects, however progress is happening.o
```

Comment:
```suggestion
        This is not a dead project. Most of the devs are currently busy with other projects, however progress is happening.
```

Beyond that - this probably does need to be said but I'm not entirely sure this reads  the right way?

Reply:
If you have a better way of wording it, I'm fine with it

Reply:
That's the thing, I'm not entirely sure myself...
Personally, for me, it's less about being busy with other projects and more about not having a lot of time (or energy) in general